### PR TITLE
Mark todo

### PR DESCRIPTION
**Pull request by macintoshhelper**

### CODE REVIEW

**Review by jwld — CHANGES_REQUESTED** [review]
Maybe needs more comments in the markTodo function to explain what the code does (lines 66-84 specifically). Also it might be a good idea to write line 70 with the standard curly bracket syntax for readability.

File: todo.js [hunk 1]
```diff
@@ -36,10 +48,40 @@ var todoFunctions = {
...
+        //   }
+        //   return newTodo;
+        // }))
+        return todos.map(function(todo){
+            // var newTodo = utilties.clone(todo);
+            var todoCopy = todo.constructor();
+            for (var attr in todo) {
+                if (todo.hasOwnProperty(attr)) todoCopy[attr] = todo[attr];
+            }
+
```
Comment (HilbertSpitzer) [inline todo.js]: //

**Review by m4v15 — COMMENTED** [review]
👍

**Review by jwld — APPROVED** [review]
cool

**Pull request merged**